src/main/java/com/segment/analytics/android/middlewares/mcvid/MarketingCloudClient.java

```diff
@@ -115,10 +118,12 @@ public String getVisitorID() throws MarketingCloudException, IOException {
         }
 
         @Override
-        public void idSync(String visitorId, String integrationCode, String customerId) throws MarketingCloudException, IOException {
+        public void idSync(String visitorId, String integrationCode, String customerId, MCVIDAuthState authentication)
+            throws MarketingCloudException, IOException {
             Map<String, String> parameters = new HashMap<>();
             parameters.put(VID_FIELD, visitorId);
-            parameters.put(CUSTOMER_ID_FIELD, String.format("%s%%01%s", integrationCode, customerId));
+            parameters.put(CUSTOMER_ID_FIELD, String.format(Locale.US, "%s" + SEPARATOR + "%s" + SEPARATOR + "%d",
```

Comment:
Another change in this PR is the `Locale.US` parameter to `String.format`. This seems reasonable as it would improve consistency. Just wanted to call that out fyi @ggm